cd/auto-generate/gh_issue_body_template.txt

```diff
@@ -0,0 +1,23 @@
+### $ISSUE_TITLE
+
+#### output for \`make build-controller\`:
+\`\`\`
+$MAKE_BUILD_OUTPUT
+\`\`\`
+
+#### Error output for \`make build-controller\`:
+\`\`\`
+$MAKE_BUILD_ERROR_OUTPUT
+\`\`\`
+
+
+#### Steps for closing this issue:
+1. Successfully generate the service controller locally using latest release of
+\`aws-controllers-k8s/code-generator\`.
+2. Update \`go.mod\` file to update \`aws-controllers-k8s/runtime\` to $ACK_RUNTIME_VERSION
```

Comment:
These steps are out of order? We should ask that they attempt to update go.mod and then fix the issues